src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,165 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
```

Comment:
Don't forget about access modifiers. Fields should be private to prevent direct access from outside the class.

Reply:
They are private!!!